Content.Server/Wagging/WaggingSystem.cs

```diff
@@ -65,6 +65,8 @@ public bool TryToggleWagging(EntityUid uid, WaggingComponent? wagging = null, Hu
 
         wagging.Wagging = !wagging.Wagging;
 
+        _actions.SetToggled(wagging.ActionEntity, wagging.Wagging);
```

Comment:
Tail Wagging needs to go into a separate PR

Reply:
TailWagging changes are extremly mini and in all honest would annoy me to move it to a different PR.